client/src/components/OnboardUser.jsx

```diff
@@ -145,40 +145,23 @@ const OnboardUser = props => {
               </Typography>
             </Grid>
             <Grid item>
-              <ButtonGroup>
-                <button
-                  type="button"
-                  onClick={e => {
-                    e.preventDefault();
-                    setFormState({
-                      ...formState,
-                      gender: 'male',
-                    });
-                  }}
-                  className={classes.buttonMale}
-                  style={
-                    formState.gender === 'male' ? { backgroundColor: '#82a4f8', outline: 'none', color: 'wheat' } : {}
-                  }
-                >
-                  Male
-                </button>
-                <button
-                  type="button"
-                  onClick={e => {
-                    e.preventDefault();
-                    setFormState({
-                      ...formState,
-                      gender: 'female',
-                    });
-                  }}
-                  className={classes.buttonFemale}
-                  style={
-                    formState.gender === 'female' ? { backgroundColor: '#82a4f8', outline: 'none', color: 'wheat' } : {}
-                  }
+              <FormControl className={classes.formControl}>
+                <Select
+                  className={classes.inputField}
+                  name="gender"
+                  value={formState.gender}
+                  onChange={handleChange}
+                  displayEmpty
+                  style={formState.gender !== '' ? { backgroundColor: '#82a4f8', outline: 'none', color: 'wheat' } : {}}
                 >
-                  Female
-                </button>
-              </ButtonGroup>
+                  <MenuItem value="" disabled>
```

Comment:
Do you need to set `value` here at all?

Reply:
This does need to be here in order for the "click here" text to show.